benchmarks/base.Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG PYTHON_VERSION=3.9-slim-buster
+
+FROM python:${PYTHON_VERSION} as python
+
+ARG BENCHMARK=sample
+ARG ARTIFACTS=/artifacts
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+WORKDIR /app
+
+# Install required system dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y \
+  curl \
+  git \
+  # cleaning up unused files
+  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
+  && rm -rf /var/lib/apt/lists/*
+
+# sirun
+RUN curl -sL https://github.com/DataDog/sirun/releases/download/v0.1.8/sirun-v0.1.8-x86_64-unknown-linux-gnu.tar.gz | tar zxf - -C /tmp && mv /tmp/sirun /usr/bin
+
+COPY ./common/entrypoint /app/
+COPY ./common/benchmark /app/
+COPY ./common/meta.yaml /app/
```

Comment:
Doesn't look like this file is checked-in

```
 => ERROR [ 7/10] COPY ./common/meta.yaml /app/                                                                                                              0.0s
------
 > [ 7/10] COPY ./common/meta.yaml /app/:
------
failed to compute cache key: "/common/meta.yaml" not found: not found
```

Reply:
Should this be `sample/meta.yaml`?